python/paddle/fluid/contrib/slim/tests/QAT_mkldnn_int8_readme.md

```diff
@@ -64,7 +66,7 @@ You can refer to the unit test in [test_quantization_mkldnn_pass.py](test_quanti
 >**III. QAT2.0 MKL-DNN C-API Performance on Intel(R) Xeon(R) Gold 6271**
 
 |     Model    | FP32 Optimized Throughput (images/s) | INT8 QAT Throughput(images/s) | Ratio(INT8/FP32) |
-|:------------:|:------------------------------------:|:-----------------------------:|:----------------:| 
+|:------------:|:------------------------------------:|:-----------------------------:|:----------------:|
 | MobileNet-V1 |                 73.98                |             227.73            |       3.08       |
 | MobileNet-V2 |                 86.59                |             206.74            |       2.39       |
 |   ResNet101  |                 7.15                 |             26.69             |       3.73       |
```

Comment:
Please append the performance of Ernie to this table.

Reply:
For now we'll provide ours, and if necessary it will be updated upon request of QA team.

Reply:
It's done